lib/plugins/aws/package/compile/events/sqs/index.js

```diff
@@ -45,10 +45,13 @@ class AwsCompileSQSEvents {
             let EventSourceArn;
             let BatchSize = 10;
             let Enabled = true;
+            let MaximumBatchingWindowInSeconds = 0;
 
             if (typeof event.sqs === 'object') {
               EventSourceArn = event.sqs.arn;
               BatchSize = event.sqs.batchSize || BatchSize;
+              MaximumBatchingWindowInSeconds =
+                event.sqs.maximumBatchingWindowInSeconds || MaximumBatchingWindowInSeconds;
```

Comment:
We need to define this option in config schema

Reply:
Thank you! Fixed by https://github.com/serverless/serverless/commit/6c54c629bfa2638564a164c23ec35ce80ebf43bc